clouddriver-api/src/main/java/com/netflix/spinnaker/clouddriver/orchestration/ApplicationContextAtomicOperationsRegistry.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2021 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.clouddriver.orchestration;
+
+import com.netflix.spinnaker.clouddriver.deploy.DescriptionValidator;
+import com.netflix.spinnaker.kork.annotations.Alpha;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.ApplicationContext;
+
+// TODO JAVADOC
+
+/**
+ * This class is to support older naming convention for operations. Do NOT use this for new
+ * operations - this is just used until older operations are migrated to new scheme
+ */
+@Alpha
+public class ApplicationContextAtomicOperationsRegistry implements AtomicOperationsRegistry {
```

Comment:
Are you still planning on subclassing this in your plugin (e.g., in `OpaAtomicOperationsRegistry`)? This class is tough because it's injecting `ApplicationContext`, which is why you need to depend on `spring-context`.

I wonder if this is the right way to get your `OpaDeployDescriptionValidator` into Clouddriver. It seems like there should be a way to have multiple description validators for a single operation description (or maybe a global description validator - that seems more like what policy engine is doing).